### PR TITLE
Enhancement/9135 selection panel 8 selection

### PR DESCRIPTION
**Pull request by zutigrm**
## Summary

Addresses issue:

- #9135

## PR Author Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 5.2 and PHP 7.4.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [x] Run the code.
- [x] Ensure the acceptance criteria are satisfied.
- [x] Reassess the implementation with the IB.
- [x] Ensure no unrelated changes are included.
- [ ] Ensure CI checks pass.
- [x] Check Storybook where applicable.
- [x] Ensure there is a QA Brief.
- [x] Ensure there are no unexpected significant changes to file sizes.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

**Comment by 10upsimon** [comment]
Passing this and moving to MR, @zutigrm and I discussed the failing VRT's and they've been fixed in other PR's, this was just not aligned with develop at the time.

**Review by 10upsimon — APPROVED** [review]

File: assets/js/components/KeyMetrics/MetricsSelectionPanel/Footer.js [hunk 1]
```diff
@@ -60,6 +61,9 @@ export default function Footer( {
 	onNavigationToOAuthURL = () => {},
 } ) {
 	const viewContext = useViewContext();
+	const isConversionReportingEnabled = isFeatureEnabled(
+		'conversionReporting'
+	);
```
Comment (tofumatt) [inline assets/js/components/KeyMetrics/MetricsSelectionPanel/Footer.js:66]: Why isn't this using the `useFeature` feature flag? 🤔

File: assets/js/googlesitekit/widgets/datastore/widgets.js [hunk 2]
```diff
@@ -71,17 +71,19 @@ export const actions = {
...
+	 * @param {number}                [settings.priority]          Optional. Widget's priority for ordering (lower number is higher priority, like WordPress hooks). Default is: 10.
+	 * @param {string|Array.<string>} [settings.width]             Optional. Widget's maximum width to occupy. Default is: "quarter". One of: "quarter", "half", "full".
+	 * @param {boolean}               [settings.wrapWidget]        Optional. Whether to wrap the component with the <Widget> wrapper. Default is: true.
+	 * @param {string|Array.<string>} [settings.modules]           Optional. Widget's associated modules.
+	 * @param {Function}              [settings.isActive]          Optional. Callback function to determine if the widget is active.
+	 * @param {Function}              [settings.isPreloaded]       Optional. Callback function to determine if the widget should be preloaded if not active (requires isActive).
+	 * @param {string|Array.<string>} [settings.hideOnBreakpoints] Optional. Hide widget on selected breakpoints. One of: "BREAKPOINT_SMALL", "BREAKPOINT_TABLET", "BREAKPOINT_DESKTOP", "BREAKPOINT_XLARGE".
```
Comment (tofumatt) [inline assets/js/googlesitekit/widgets/datastore/widgets.js:86]: The documentation for this param is a bit off, but let's also change it to only be an array. I think it gets a bit confusing when the argument is plural (eg. implied to be an array) but it also accepts a single value.

```suggestion
	 * @param {Array.<string>} [settings.hideOnBreakpoints] Optional. Hide widget on selected breakpoints. Array with any of: `BREAKPOINT_SMALL`, `BREAKPOINT_TABLET`, `BREAKPOINT_DESKTOP`, `BREAKPOINT_XLARGE`.
```

File: assets/js/googlesitekit/widgets/index.js [hunk 3]
```diff
@@ -82,14 +82,15 @@ export function createWidgets( registry ) {
...
-		 * @param {(string|Array)}        [widgetAreaSlugs]     Optional. Widget area slug(s).
+		 * @param {string}                slug                         Widget's slug.
+		 * @param {Object}                settings                     Widget's settings.
+		 * @param {WPComponent}           settings.Component           React component used to display the contents of this widget.
+		 * @param {number}                [settings.priority]          Optional. Widget's priority for ordering (lower number is higher priority, like WordPress hooks). Default is: 10.
+		 * @param {string|Array.<string>} [settings.width]             Optional. Widget's maximum width to occupy. Default is: "quarter". One of: "quarter", "half", "full".
+		 * @param {boolean}               [settings.wrapWidget]        Optional. Whether to wrap the component with the <Widget> wrapper. Default is: true.
+		 * @param {string|Array.<string>} [settings.modules]           Optional. Widget's associated moduels.
+		 * @param {string|Array.<string>} [settings.hideOnBreakpoints] Optional. Hide widget on selected breakpoints. One of: "BREAKPOINT_SMALL", "BREAKPOINT_TABLET", "BREAKPOINT_DESKTOP", "BREAKPOINT_XLARGE".
```
Comment (tofumatt) [inline assets/js/googlesitekit/widgets/index.js:92]: Let's update this too to just be an array.

Comment (tofumatt) [reply assets/js/googlesitekit/widgets/index.js:92]: In fact, it doesn't even look like this _does_ accept a raw string: https://github.com/google/site-kit-wp/blob/a2a949c4284f747680deb5b7c7c4fb103400b346/assets/js/googlesitekit/widgets/components/WidgetRenderer.js#L71

So we should definitely update the docs here.

**Review by tofumatt — CHANGES_REQUESTED** [review]
Functionality looks good, but there's a few things to fix in the code before merging. 👍🏻

File: assets/js/components/KeyMetrics/MetricsSelectionPanel/Footer.js (hunk 1 above)
Comment (zutigrm) [reply assets/js/components/KeyMetrics/MetricsSelectionPanel/Footer.js:66]: Oh indeed, I mistakenly used `isFeatureEnabled`. Fixed

**Comment by zutigrm** [comment]
Thanks @tofumatt PR updated

**Pull request merged**